### ISSUE TITLE
disable the build hotkey? or turn off build completely?

### ISSUE THREAD

**Issue by djangojack** [Microsoft/TypeScript-Sublime-Plugin#483]
On OSX, whenever I hit command+b instead of command+v, sublime builds all my js files, which is a huge pain because I have a separate build process, which builds everything in a separate directory.

How can I prevent this plugin from building anything ever?

**Comment by kyranjamie** [comment]
Agreed, this is super annoying. I'm not sure how many people utilise this option, but I can't imagine many. 

Building all *.js files into your src directory can be dangerous also. All newly written code in git hidden among hundreds of compiled files. `git clean -f` anyone 👀

**Comment by kyleconkright** [comment]
Any update here? The v and b key are too close and using a build process is pretty common. Common enough that ide compiling should be an option. In my opinion of course.

**Comment by evanjmg** [comment]
+1 - This has cost me hours of development

**Comment by Jadamso** [comment]
+1

**Comment by afrieder** [comment]
+1

**Comment by elcommie** [comment]
+1.  I'm having to do a lot of `git clean -f` and hope I haven't created any new files

**Comment by kronik3r** [comment]
Hello everyone, 

This is pretty old, but it's still a valid issue. It's actually a `Sublime Text` issue that you can't disable the build systems, but for the sake of this specific problem. I found a workaround by binding the shortcut to a harmless command. (I associated it with `cancel_build`)

I am using a Mac and the shortcut to `build` is `cmd+b`, so I added this to my `Default (OSX).sublime-keymap`:
```json
{ "keys": ["super+b"], "command": "cancel_build" },
```

PS: I am tagging you here @elcommie being the last one to comment on this thread, this might help.

**Comment by elcommie** [comment]
Thanks @kronik3r,

That's really a life saver.
